tests/tests.py

```diff
@@ -123,11 +123,68 @@ def test040_annotate(self):
 
         del_resp.close()
 
-    def test050_delete(self):
+    def test050_trackhub(self):
+        print "Test#5 Trackhub Creation Test"
+        driver = self.driver
+        driver.get("http://127.0.0.1:8080/")
+        self.login(driver)
+
+        # make sure that we are logged in
+        wait = WebDriverWait(driver, 60)
+        assert wait.until(
+            EC.element_to_be_clickable((By.ID, "signout"))).is_displayed()
+        driver.get("http://127.0.0.1:8080/trackhub_init/seganntest2@gmail.com/")
+        shortname_field = driver.find_element_by_id("id_short_label")
+        shortname_field.send_keys("test1")
+        longname_field = driver.find_element_by_id("id_long_label")
+        longname_field.send_keys("test1long")
+        db_field = driver.find_element_by_id("id_db")
+        db_field.send_keys("hg19")
+        profile_box = driver.find_element_by_xpath("//input[@value='ES0004']")
+        profile_box.click()
+        submit_button = driver.find_element_by_id("submit_button")
+        submit_button.click()
+        sleep(10)
+        if ("Trackhub data for test1" in driver.page_source):
+            assert True
```

Comment:
nice, glad to see that you added a test for the track hub functionality that you implemented

you may also want to check for the presence of the profile name, ES0004, on the web page

also it would be good test track hubs with multiple profiles (here there is only one profile in the track hub, right?)